### DISCUSSION TITLE
Piston movement

### DISCUSSION THREAD

**Discussion by GoncaloT** [GriefPrevention/GriefPrevention#1242] [Q&A] [answered]
Hey, i can´t use my pistons cause of this plugin and i don´t know how to desactivate it and permit the piston movement to all, i have right here the problem i just don´t know the config name to desactivate it
https://prnt.sc/ygq63v

**Comment by Jikoo** [comment, marked as answer]
https://github.com/TechFortress/GriefPrevention/wiki/Setup-and-Configuration#pistonmovement

**Reply by GoncaloT** [reply]
omg omg ty so muchhhhh!
